addons/sale/static/src/js/product_configurator_dialog/product_configurator_dialog.js

```diff
@@ -65,6 +76,8 @@ export class ProductConfiguratorDialog extends Component {
                 );
             }
             this._checkExclusions(this.state.products[0]);
+            // Use the currency id retrieved from the server if none was provided in the props.
+            this.currency.id ??= currency_id;
```

Comment:
Do you store anything else than `id` in `this.env.currency`?

Reply:
Nope, only the id, but I need the wrapper object so that this change gets propagated to the env.

Reply:
Oh smart, so no need for getters anymore